Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -70,6 +70,25 @@ class DatabaseTest extends TestCase
     private $database;
     private $session;
 
+    private static $databaseName;
+    private static $sessionName;
+    private $getDatabaseArgs;
```

Comment:
I'm not sure we need to worry about DRY rules in unit tests. I'd prefer each test to define its assertions explicitly. This is a clever idea, but I think it costs us readability. Same goes for the other assertions from 76-78.

Reply:
Well, I hate copypasting, but I love readability. All right, I'll change it.